### ISSUE TITLE
v0.3.2 High CPU (no GPU offload?)

### ISSUE THREAD

**Issue by nPHYN1T3** [lmstudio-ai/lmstudio-bug-tracker#102]
I'm noticing with v0.3.2 my CPU is getting slaughtered. The UI revamp is worse than the previous iteration with GPU offload now hidden on "My Models" page but even with all the layers assigned to GPU my CPU has all the work.

*record scratch* I just went and fiddled with things again and now it looks like it's cross loading across all GPU's and CPU. Which seems like settings are not "taking" correctly. However CPU use is high just with LMStudio open and idle compared to 0.2.8.

As I was basically once again sliding to 0, slide to max just to test the model also seemed to loose everything from today. As I asked a new question to see where the loading went it says there is a miscommunication and tries to answer something from like a week ago. What? Magic persistence?!

It's starting to look like I should just come back to this in a few years...

**Comment by smirgol** [comment]
I'm not sure if that helps, but I had a similar problem where GPU acceleration just wasn't available anymore on my AMD RX7900XTX. I finally found out why - by default CPU mode is enabled and it will not even show any GPU info, like if none exists. That can be changed in a rather hidden setting:

1. You need to switch the GUI to "Developer" mode - on the bottom left of the GUI there should be three buttons, one being "Developer".
2. Then on the left there should be a new green icon with tooltip "Developer". Click that.
3. In the main window at the top, right of the "LM Studio" there should be two buttons "Local Server" and "LM Runtimes". Click "LM Runtimes".
4. Now, on the right side there is "Runtime Preferences" and below that "Configure Runtimes".
5. Switch the runtime for "GGUF" from "CPU llama.cpp" to "Vulkan llama.cpp".
6. There you go.

Maybe that helps someone.
Cheers!

**Comment by nPHYN1T3** [comment]
I already outlined the moved/hidden GPU settings in the OP. Using the new GPU offload UI the workload is still dumped on the CPU.

**Comment by smirgol** [comment]
True, but note that setting the `GPU Offload` setting on the `My Models` tab won't do anything if LM-Studio is running the cpu-only llama, which was my issue and which can be fixed as described above.

**Comment by aqasem81** [comment]
Same issue here, I'm running this on Windows 11 and have iGPU Arc and Nvidia 4060 the load by default is on the CPU and when I tried the GPU offload setting nothing changed.

**Comment by Bo-The-Lab** [comment]
I was having the same issue: LM Studio only running models on my Laptops CPU. While I was testing the CPU Offload setting, I noticed the drop-down box above all the driver options. 

> 2. Select the preferred graphics processor.

Once I chose the "High Performance NVIDIA Processor", LM Studio would use the GPU.  Looks like the settings for the GPU mux is somehow getting in the way.  I always assumed the mux was in the video output section, after the frame buffers.

Now LM Studio is directed to the correct GPU, I've observed some other problem with the model I'm trying out this week and/or LM Studio settings.  I get one of two behaviours:
* The GPU runs at 100% with no output at all, I assume the model has crashed somehow.
* There is a burst of GPU activity, about 70% for up to 30 seconds, then all output takes place on the CPU.  Subsquent generations have a shorter run time of this process, jumping to the CPU faster.
Fast enough for testing and messing around with, but makes the GPU feel like a waste of money. :shrug:
